lambdatimes/src/components/Content/Tabs.js

```diff
@@ -1,10 +1,19 @@
-import React from 'react';
-import Tab from './Tab';
+import React from "react";
+import Tab from "./Tab";
+import PropTypes from "prop-types";
+
 const Tabs = props => {
   return (
     <div className="tabs">
       <div className="topics">
         <span className="title">TRENDING TOPICS:</span>
+        {props.tabs.map(tab => (
+          <Tab
+            selectTabHandler={props.selectTabHandler}
```

Comment:
In the console it gives an error for missing key. `Warning: Each child in a list should have a unique "key" prop.`

So you just need to add `key={tab}` and also on Cards.js same issue with missing key sp add `key={card.headline}`